### DISCUSSION TITLE
Navigation in Background

### DISCUSSION THREAD

**Discussion by danwhite-ipc** [react-navigation/react-navigation#9610] [Q&A]
Is there anyway to improve handling of a navigation event triggering while the app is in the background. Currently I have a call (voip) handling screen but if the call is destroyed while the app is in the background when I next open the app I see the transition between screens. Any ideas on improving the experience?

**Comment by raajnadar** [comment]
You should use navigation ref to navigate directly when you move from background to foreground.
